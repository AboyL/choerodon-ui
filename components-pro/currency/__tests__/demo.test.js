import demoTest from '../../../tests/shared/demoTest';

demoTest('currency-pro');
