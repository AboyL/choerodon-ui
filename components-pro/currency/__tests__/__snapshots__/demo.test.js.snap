// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders ./components-pro/currency/demo/basic.md correctly 1`] = `
<div>
  <div
    class="c7n-row"
    style="margin-left:-5px;margin-right:-5px"
  >
    <div
      class="c7n-col-12"
      style="padding-left:5px;padding-right:5px"
    >
      <span
        class="c7n-pro-input-number-wrapper"
      >
        <label>
          <input
            autocomplete="off"
            class="c7n-pro-input-number"
            type="text"
            value="10,000.00"
          />
        </label>
      </span>
    </div>
    <div
      class="c7n-col-12"
      style="padding-left:5px;padding-right:5px"
    >
      <span
        class="c7n-pro-input-number-wrapper"
      >
        <label>
          <input
            autocomplete="off"
            class="c7n-pro-input-number"
            currency="CNY"
            type="text"
            value="Â¥10,000.00"
          />
        </label>
      </span>
    </div>
  </div>
</div>
`;

exports[`renders ./components-pro/currency/demo/controlled.md correctly 1`] = `
<span
  class="c7n-pro-input-number-wrapper"
>
  <label>
    <input
      autocomplete="off"
      class="c7n-pro-input-number"
      type="text"
      value="100.00"
    />
  </label>
</span>
`;

exports[`renders ./components-pro/currency/demo/dataset.md correctly 1`] = `
<span
  class="c7n-pro-input-number-wrapper c7n-pro-input-number-required"
>
  <label>
    <input
      autocomplete="off"
      class="c7n-pro-input-number"
      name="money"
      type="text"
      value="US$100,000,000,000,000.00"
    />
  </label>
</span>
`;
